server/src/api/observation/observation.controller.ts

```diff
@@ -16,14 +16,15 @@ class ObservationController {
 }
 
 async function createObservation(req: ApiRequest<EdipiParam, ObservationApiModel>) {
+  const body = req.body;
+  console.log(`Creating observation ${JSON.stringify(body)}`);
   // getManager().query() does not return the inserted row
-  console.log(`Creating observation ${JSON.stringify(req.body)}`);
+  const timeStamp = timestampColumnTransformer.to(body.timestamp);
   await getManager()
     .query(
-      'INSERT INTO public.observation(document_id, edipi, timestamp, unit_id, unit, type_id, type_org)'
-      + 'VALUES ($1, $2, $3, $4, $5, $6, $7)',
-      [req.body.documentId, req.body.edipi, timestampColumnTransformer.to(req.body.timestamp),
-        req.body.unitId, req.body.unit, req.body.typeId, req.body.orgId],
+      'INSERT INTO public.observation(document_id, edipi, timestamp, unit, type_id, reporting_group)'
+      + 'VALUES ($1, $2, $3, $4, $5, $6)',
+      [body.documentId, body.edipi, timeStamp, body.unit, body.typeId, body.reportingGroup],
     );
 }
```

Comment:
I think you can simplify this function down to this...

```typescript
async function createObservation(req: ApiRequest<EdipiParam, ObservationApiModel>) {
  Log.info('Creating observation', req.body);
  const observation = Observation.create(req.body);
  return observation.save();
}
```

NOTE: We're using `Log.info` in place of `console.log` throughout the codebase so that we have more control over when/how logs are displayed. For example, when using `Log`, all logs will be filtered out of test output by default.

Reply:
Done, please review.